gemrb/core/GUI/TextSystem/GemMarkup.cpp

```diff
@@ -164,9 +202,14 @@ GemMarkupParser::ParseMarkupStringIntoContainer(const String& text, TextContaine
 				}
 				break;
 			case INT:
-				if (*it == L']') {
+				if (*it == u']') {
+					std::wstring_convert<std::codecvt_utf16<wchar_t, 0x10ffff, std::little_endian>, wchar_t> conv;
+					std::wstring wToken = conv.from_bytes(
```

Comment:
seems like we should be able to do this without `std::wstring`, no?

Reply:
What's your idea? I did not found anything related to `std::u16string` yet.